utilities/virt.py

```diff
@@ -1994,7 +1995,7 @@ def vm_instance_from_template(
         existing_data_volume=existing_data_volume,
         vm_dict=params.get("vm_dict"),
         cpu_cores=params.get("cpu_cores"),
-        cpu_threads=params.get("cpu_threads"),
+        cpu_threads=1 if is_cluster_s390x() else params.get("cpu_threads"),
```

Comment:
I don't like the idea to update it here in the such common place

We don't have many places where we use cpu_threads more than 1, so I would prefer if you use fixture `is_s390x_cluster` in necessary fixtures instead of introducing this global w/a

Reply:
there is a big chance that these `~10 places` are dropped down to only 2-3 fixtures
also, some of these tests potentially can be not usable in the s390x clusters

so you can update it only in necessary fixtures, please do not do that globally